<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SEARCH ENGINE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresource.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EXIT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HISTORY">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
    </property>
    <property name="text">
     <string>Lịch sử tìm kiếm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SEARCH">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);
cursor: pointer;</string>
    </property>
    <property name="text">
     <string>Tìm kiếm</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="EDIT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
    </property>
    <property name="text">
     <string>Chỉnh sửa dữ liệu</string>
    </property>
   </widget>
   <widget class="QLabel" name="AUTHOR">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>570</y>
      <width>301</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Đồ án môn học&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Kỹ thuật lập trình&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ABOUT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
    </property>
    <property name="text">
     <string>Giới thiệu</string>
    </property>
   </widget>
   <widget class="QLabel" name="appName">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>-4</y>
      <width>1011</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SVN-Revolution</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/logo4.png);
background-size: cover;
background-position: center;
background-repeat: no-repeat;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>751</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="aboutTab">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>681</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#333333;&quot;&gt;Ứng dụng có chức năng chính là trả về những file có trong dữ liệu mà nó có chứa nội dung liên quan đến thông tin tìm kiếm. Để mở một file nào đó trong kết quả tìm kiếm, hãy double click vào dòng chứa kết quả tương ứng.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#333333;&quot;&gt;Người dùng có thể thêm/xóa những file văn bản của mình vào dữ liệu của chương trình thông qua mục &amp;quot;Chỉnh sửa dữ liệu&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#0000ff;&quot;&gt;Thông tin nhóm tác giả:&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; color:#333333;&quot;&gt;Hình ảnh chỉ mang tính chất vui vẻ ^^&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>215</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/vu.png);
background-position: center;
border: 1px solid rgb(176, 176, 176);
border-radius: 40%;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>435</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000ff;&quot;&gt;Hoàng Trọng Vũ&lt;br/&gt;MSSV: 20120025&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>435</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000ff;&quot;&gt;Nhan Gia Khâm&lt;br/&gt;MSSV: 20120504&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>215</y>
        <width>256</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/kham.png);
background-position: center;
border-radius: 40%;
border: 1px solid rgb(176, 176, 176);</string>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>530</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#0000ff;&quot;&gt;Trường Đại học Khoa học Tự nhiên - ĐHQG TPHCM&lt;br/&gt;CNTN2020 - Khoa Công nghệ Thông tin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>503</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#0000ff;&quot;&gt;GV hướng dẫn: TS. Võ Hoài Việt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchTab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QPushButton" name="searchBtn">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>15</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
      </property>
      <property name="text">
       <string>Tìm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(93, 93, 93);
padding-left: 10px;
border-radius: 10px;</string>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Tìm kiếm gì đó...</string>
      </property>
     </widget>
     <widget class="QListWidget" name="searchResult">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>90</y>
        <width>711</width>
        <height>467</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;
border: 1px solid rgb(93, 93, 93);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="searchStatus">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>56</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showMore">
      <property name="geometry">
       <rect>
        <x>286</x>
        <y>562</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);
cursor: pointer;</string>
      </property>
      <property name="text">
       <string>Hiển thị thêm kết quả</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="historyTab">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QListWidget" name="historyResult">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>48</y>
        <width>711</width>
        <height>495</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;
border: 1px solid rgb(93, 93, 93);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0000ff;&quot;&gt;Lịch sử tìm kiếm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="historyDeleteSome">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>551</y>
        <width>161</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);
cursor: pointer;</string>
      </property>
      <property name="text">
       <string>Xóa mục đã chọn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="historyDeleteAll">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>551</y>
        <width>161</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);
cursor: pointer;</string>
      </property>
      <property name="text">
       <string>Xóa tất cả</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="editTab">
     <attribute name="title">
      <string>Edit DB</string>
     </attribute>
     <widget class="QLabel" name="database">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0000ff;&quot;&gt;Chỉnh sửa tập dữ liệu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>621</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#0000ff;&quot;&gt;Thêm file:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="directFileAdded">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>140</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(93, 93, 93);
padding-left: 10px;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Nhập đường dẫn tuyệt đối đến file .txt muốn thêm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addFileBtn">
      <property name="geometry">
       <rect>
        <x>601</x>
        <y>140</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
      </property>
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>195</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#0000ff;&quot;&gt;Những file đã thêm:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delFileBtn">
      <property name="geometry">
       <rect>
        <x>326</x>
        <y>543</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid rgb(0, 55, 255);
border-radius: 15px;
color: rgb(0, 55, 255);</string>
      </property>
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
     <widget class="QLabel" name="updateNoti">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="fileOfUser">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>243</y>
        <width>668</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;
border: 1px solid rgb(93, 93, 93);</string>
      </property>
     </widget>
     <widget class="QLabel" name="addStatus">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>185</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>107</y>
        <width>701</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#333333;&quot;&gt;Chỉ nhập đường dẫn không dấu và file .txt phải được lưu dạng UTF-16LE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="myresource.qrc"/>
 </resources>
 <connections/>
</ui>
